elyra/util/cos.py

```diff
@@ -55,9 +55,10 @@ def __initialize_object_store(self):
         try:
             if not self.client.bucket_exists(self.bucket):
                 self.client.make_bucket(self.bucket)
-        except BucketAlreadyOwnedByYou as ex:
+        except BucketAlreadyOwnedByYou:
+            # Note: the check for `bucket_exists` does not work in the IBM Cos
+            # so we should not re-raise the exception
             self.log.warning("Object Storage bucket already owned by you", exc_info=True)
-            raise ex from ex
         except BucketAlreadyExists as ex:
             self.log.warning("Object Storage bucket already exists", exc_info=True)
             raise ex from ex
```

Comment:
If we're going to raise this exception then it should be logged as an Error, not a Warning.

Reply:
I see this as not an error, you call a function and the end result is to have a bucket created and owned by you. In this case, we are doing more work, as we are trying to create a bucket that already exists, but the end result is the same, which then we should not log as an error.

Reply:
I agree this is a warning situation, but why raise the error then?  This is going to raise an exception during the creation of CosClient instances.  Seems like treating it like AlreadyOwnedByYou is the way to go too.

Reply:
nvm, I was confusing with the exception we are actually changing....

Reply:
I see.   I suppose this case is only necessary for IBM COS, since we can't trust the existence check, and if they don't throw `BucketAlreadyOwnedByYou `, but DO throw `BucketAlreadyExists `, then that implies the bucket is NOT owned by you - is that the thought here?  

If so, I wonder if adding a clause like: `"Object Storage bucket already in use by another user"` would be more helpful - because if it already exists and is owned by me - then why fail the `CosClient` creation and prevent me from using my bucket?

Do we know if that's the case?  Seems like an odd way to let someone know the operation is not allowed.